.github/workflows/build_with_conda.yml

```diff
@@ -16,6 +16,12 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
+            cpp_tests: "1"
+            python_tests: "0"
```

Comment:
The matrix variables support `true`/`false` you know

Reply:
but we want to assign the value to an env variable `ARTICDB_BUILD_CPP_TESTS`

Reply:
I am actually a bit annoyed we need to do this GitHub Action -> `setup.py` -> cmake dance.

It might be simpler to check this environment variable in CMake directly (without changing `setup.py`), which will also give you the option to pass `"true"` in addition to `"1"`.

Reply:
> I am actually a bit annoyed we need to do this GitHub Action -> `setup.py` -> cmake dance.

in particular when only building c++ tests